app/javascript/elm/src/FixedSessionFilters.elm

```diff
@@ -44,44 +44,33 @@ init flags =
 
 
 type Msg
-    = UpdateTagsSearch String
-    | AddTag String
-    | RemoveTag String
-    | UpdateProfileSearch String
-    | AddProfile String
-    | RemoveProfile String
+    = TagsLabels LabelsInput.Msg
+    | ProfileLabels LabelsInput.Msg
 
 
 update : Msg -> Model -> ( Model, Cmd Msg )
 update msg model =
     case msg of
-        UpdateTagsSearch content ->
-            ( { model | tags = Labels.updateCandidate model.tags content }, Cmd.none )
+        TagsLabels subMsg ->
+            updateLabels subMsg model.tags Ports.updateTags TagsLabels (\tags -> { model | tags = tags })
```

Comment:
This should make easier to spot a problem where we use a "profile" thing instead of a "tag" thing (eg `ProfileLabels` instead of `TagLabels`). In other words, the differences are here and the noise is in `updateLabels`. We could one day refactor in a way that the types make tags and profiles different